Application/Frontend/components/FriendsColumn/Fetch_chat.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react';
+
+function ChatOwnerFinder() {
+  const [channelKey, setChannelKey] = useState('');
+  const [owner, setOwner] = useState('');
+  const [error, setError] = useState('');
+
+  const fetchChatOwner = async () => {
+    setError(''); // Reset error message
+    try {
+      const response = await fetch('/api/get_chat', { // Adjust with your API endpoint
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({ channelKey }),
```

Comment:
Great use of the correct primary key in our collection. Although you intake a parameter of any name within the body of the HTTP request, using the same variable name as the primary key attribute in the collection conforms to best-practice. Not doing that would be a code smell. Great work!